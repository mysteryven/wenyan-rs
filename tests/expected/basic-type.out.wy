2
false
true
2 false true
