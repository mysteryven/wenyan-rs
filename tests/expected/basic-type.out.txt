2
false
true
2 false true
金风玉露一相逢，便胜却人间无数
金风玉露一相逢，便胜却人间无数
